internal/graph/resolvers/event.go

```diff
@@ -33,6 +34,21 @@ const (
 	errListEvents    = "cannot list events"
 	errGetInvolved   = "cannot get involved resource"
 	errModelInvolved = "cannot model involved resource"
+
+	// maxGlobalEvents is the maximal number of events we return in a single
+	// request, ordered by time. This is to prevent the client from requesting
+	// all 70k events or whatever the number is.
+	maxGlobalEvents = 2000
```

Comment:
Do these values seem like things we would want to be configurable within the environment (using environment variables or args)?

Reply:
will add those tomorrow.